### PR TITLE
Form fields preview state

### PR DESCRIPTION
**Pull request by zdlm**
@wiredearp @zdlm @sampi @GeorgeIuriet 

Fixes issue [Forms] Form fields preview state #230

### CODE REVIEW

File: src/runtime/js/ts.ui/forms/forms-gui@tradeshift.com/spirits/ts.ui.FieldSetSpirit.js [hunk 1]
```diff
@@ -38,6 +38,14 @@ ts.ui.FieldSetSpirit = (function() {
 			}
 		},
 
+		onasync: function() {
+			this.super.onasync();
+			var form = this.dom.parent(ts.ui.FormSpirit);
+			if (form && form.css.contains(ts.ui.CLASS_PREVIEW)) {
+				this.element.setAttribute('disabled', 'disabled');
```
Comment (sampi) [inline src/runtime/js/ts.ui/forms/forms-gui@tradeshift.com/spirits/ts.ui.FieldSetSpirit.js]: didn't we want this to be `readonly` ?
the text should still be selectable somehow

File: src/runtime/js/ts.ui/forms/forms-gui@tradeshift.com/spirits/ts.ui.FormSpirit.js [hunk 2]
```diff
@@ -48,7 +48,16 @@ ts.ui.FormSpirit = (function() {
...
 			 */
-			_model: null
+			_model: null,
+
+			/**
+			* Form fields preview state
+			* @param @optional {boolean} ispreview
+			*/
+			preview: function(ispreview){
+				ispreview = ispreview === 'true' ? true : false;
```
Comment (sampi) [inline src/runtime/js/ts.ui/forms/forms-gui@tradeshift.com/spirits/ts.ui.FormSpirit.js]: `ispreview = ispreview === 'true'` is the same

File: src/runtime/less/ts-forms.less [hunk 3]
```diff
@@ -863,3 +864,26 @@
 		}
 	}
 }
+
+// Preview .....................................................................
+
+.ts-form {
+	&.ts-preview {
+		.ts-datelabel,.ts-fakelabel {
```
Comment (sampi) [inline src/runtime/less/ts-forms.less]: space missing, we should use prettier for the less code as well.

Comment (sampi) [inline src/runtime/less/ts-forms.less]: also, the selectors should be a new lines

File: src/runtime/less/ts-forms.less [hunk 4]
```diff
@@ -863,3 +864,26 @@
...
+			&:before {
+				display: none !important;
+			}
+		}
+		.ts-field.ts-text, .ts-field.ts-select {
```
Comment (sampi) [inline src/runtime/less/ts-forms.less]: these selectors should be on new lines

File: docs/dist/appendix/autolayout/index.html [hunk 5]
```diff
@@ -34,7 +34,7 @@
 <body data-ts="Spirit" id="body">
     <script class="gui-script" data-ts.scriptid="edbml.$iSNtu1">
         edbml.declare("edbml.$iSNtu1").as(function $edbml(
-            /*``*/
+            /**/
```
Comment (sampi) [inline docs/dist/appendix/autolayout/index.html]: did you do a `yarn install` to make sure everything is the latest?

File: src/runtime/js/ts.ui/forms/forms-gui@tradeshift.com/spirits/ts.ui.FieldSetSpirit.js (hunk 1 above)
Comment (zdlm) [inline src/runtime/js/ts.ui/forms/forms-gui@tradeshift.com/spirits/ts.ui.FieldSetSpirit.js]: You can select text on `disabled` not on `readonly`

**Pull request merged**